hypershift-operator/main.go

```diff
@@ -434,3 +439,148 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 	log.Info("starting manager")
 	return mgr.Start(ctx)
 }
+
+func NewSetupCommand() *cobra.Command {
```

Comment:
This should be NewInitCommand to keep naming consistent

Reply:
Thanks for letting me know. Using `NewInitCommand` as the function signature now.